src/js/components/Playlists/Playlists.react.js

```diff
@@ -38,14 +39,17 @@ export default class Playlists extends Component {
 
         if(this.props.playlists.length === 0) {
             return (
-                <FullViewMessage message="You haven't created any playlist yet">
-                    <a onClick={ this.createPlaylist }> create one now </a>
+                <FullViewMessage>
+                    <p>You haven't created any playlist yet</p>
+                    <FullViewSubMessage>
+                        <a onClick={ this.createPlaylist }>create one now</a>
+                    </FullViewSubMessage>
                 </FullViewMessage>
```

Comment:
Componentize is ok, but here it's definetly too much. Just do a FullViewMessage:

<FullViewMessage>
    <p>blabla</p>
    <p className='sub-message'>blabla</p>
</FullViewMessage>

or don't do it at all. It's perfectly fine.